frontend/services/RestConnection.ts

```diff
@@ -122,6 +123,47 @@ export default class RestConnection implements IConnection {
         });
 
         return promise;
+    }
+
+    public getNotifications(): Promise<NotificationResponse[]> {
+        const url = `${Config.SERVER_URL}/notifications?u_id=${Expo.Constants.deviceId}`;
+        const promise = new Promise<NotificationResponse[]>((resolve, reject) => {
+            axios.get(url, DEFAULT_OPTIONS)
+                .then((response) => {
+                    resolve(response.data);
+                })
+                .catch((error) => {
+                    reject('Couldn\'t get the old Notifications');
+                });
+        });
+        return promise;
+    }
 
+    public markNotificationsAsRead(): Promise<void> {
+        const url = `${Config.SERVER_URL}/notifications/markAsRead?u_id=${Expo.Constants.deviceId}`;
+        const promise = new Promise<void>((resolve, reject) => {
```

Comment:
Ein Promise<void>? Warum machst du die Funktion nicht async und haust hier stattdessen ein await rein?

Reply:
Ich habe mich hier stur an allen anderen Funktionen orientiert. Ich bin mir ehrlich gesagt auch nicht sicher wie das Error-Handling mit dem reject() ohne Promise aussieht. Zudem nutzt auch deine Funktion getConversationHistory() den gleichen Promise. Ist die Frage ob wir das umbauen wollen? Falls ja hätte ich vorgeschlagen, das in einem extra PR zu machen.